block/manager.go

```diff
@@ -383,8 +384,12 @@ func (m *Manager) UpdateTargetHeight(h uint64) {
 
 // ValidateConfigWithRollappParams checks the configuration params are consistent with the params in the dymint state (e.g. DA and version)
 func (m *Manager) ValidateConfigWithRollappParams() error {
-	if version.Commit != m.State.RollappParams.Version {
-		return fmt.Errorf("binary version mismatch. rollapp param: %s binary used:%s", m.State.RollappParams.Version, version.Commit)
+	dymintVersion, err := strconv.ParseUint(version.DRSVersion, 10, 64)
```

Comment:
I think it's confusing to call it dymint version - cause it's not. it's actually the reason we did this - to not couple it with dymint.
I think `DRSVersion` is fine.

Reply:
renamed